Overview.src.html

```diff
@@ -662,6 +662,26 @@ <h4 id=terminology-headers>Headers</h4>
 <span data-anolis-spec=html>environment settings object</span>).
 
 <p>A <span title=concept-request>request</span> has an associated
+<dfn title=concept-request-reserved-client data-export data-dfn-for=request>reserved client</dfn>
+(null, an <span data-anolis-spec=html>environment</span>, or an
+<span data-anolis-spec=html>environment settings object</span>). Unless stated otherwise it is null.
+
+<p class="note no-backref">This is only used by <span>navigation requests</span> and worker requests
+(except service worker requests). It references an <span data-anolis-spec=html>environment</span>
+for a <span>navigation request</span> and an <span data-anolis-spec=html>environment settings
+object</span> for a worker request.
+
+<p>A <span title=concept-request>request</span> has an associated
+<dfn title=concept-request-target-client-id data-export data-dfn-for=request>target client id</dfn>
+(a string). Unless stated otherwise it is the empty string.
+
+<p class="note no-backref">This is only used by <span>navigation requests</span>. It has the
+<span data-anolis-spec=html>id</span> of the <span data-anolis-spec=html>target browsing
+context</span>'s <span data-anolis-spec=html>active document</span>'s
+<span data-anolis-spec=html>environment settings object</span> captured before entering the
+<span title=concept-fetch>fetch</span> algorithm.
```

Comment:
Okay, thanks for confirming. Once the HTML PR has a commit message and is landed I'll merge this with a couple of minor tweaks.